webrtc/videoroom/lib/videoroom_web/room_channel.ex

```diff
@@ -85,11 +85,24 @@ defmodule VideoRoomWeb.RoomChannel do
     {:noreply, socket}
   end
 
+  @impl true
+  def handle_info({:signal, {:replace_track, old_track_id, new_track_id}}, socket) do
```

Comment:
I think that you send this information about replacing track for each infividual endpoint (correct me if I'm wrong) so you should probably not broadcast this information (you end up with `n^2` messages instead of `n`). Simple `push` should do the job.